resources/assets/less/bem/beatmap-discussion-editor.less

```diff
@@ -50,6 +74,13 @@
     }
   }
 
+  &__inner-block-count {
```

Comment:
collides with the new insertion bar and a bit too close to the unlink button

![](https://s.myconan.net/2020-07/firefox_2020-07-31_20-01-34.png)